src/README.md

```diff
@@ -7,38 +7,6 @@ install all dependencies with `npm install`, then launch the environment with
 `npm run docker-watch`. Webpack will watch changes made to the repository and
 update the theme (+ refresh your browser) accordingly.
 
```

Comment:
Not sure why this is being removed. It might be because the new branch was missing the README.

Reply:
Seems to be the case, in this case I would probably pull in changes from README.md that Wilson made

Reply:
You mean just pull the README.md in src file from master? Or is it from a branch Wilson made?

Reply:
I think from my branch, though I would be careful of conflicts we may have between our branches.